package.json

```diff
@@ -23,7 +23,9 @@
     "react": "^18",
     "react-dom": "^18",
     "react-icons": "^5.0.1",
-    "react-markdown": "^9.0.1"
+    "react-intersection-observer": "^9.13.1",
+    "react-markdown": "^9.0.1",
```

Comment:
Do we still use these two packages?

Reply:
react-markdown, yes. 
react-intersection-observer, no.